src/utc.rs

```diff
@@ -8,236 +8,428 @@ use super::*;
     #[serde(skip_serializing_if = "Option::is_none", rename = "ScriptSpellAt")]
     pub script_spell_at: Option<NwValue<String>>,
+
+    // The user-defined script associated with the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "ScriptUserDefine")]
     pub script_user_define: Option<NwValue<String>>,
+
+    // The list of skills associated with the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "SkillList")]
     pub skill_list: Option<NwValue<Vec<Skill>>>,
+
+    // The sound set file associated with the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "SoundSetFile")]
     pub sound_set_file: Option<NwValue<i32>>,
+
+    // The list of special abilities associated with the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "SpecAbilityList")]
     pub special_ability_list: Option<NwValue<Vec<SpecialAbility>>>,
+
+    // The ID of the starting package associated with the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "StartingPackage")]
     pub starting_package: Option<NwValue<i32>>,
+
+    // The strength attribute value of the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "Str")]
     pub strength: Option<NwValue<i32>>,
+
+    // The subrace associated with the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "Subrace")]
     pub subrace: Option<NwValue<String>>,
+
+    // The tag associated with the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "Tag")]
     pub tag: Option<NwValue<String>>,
+
+    // The new tail ID associated with the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "Tail_New")]
     pub tail_new: Option<NwValue<i32>>,
+
+    // The list of templates associated with the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "TemplateList")]
     pub template_list: Option<NwValue<Vec<Template>>>,
+
+    // The template resource reference associated with the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "TemplateResRef")]
     pub template_res_ref: Option<NwValue<String>>,
+
+    // The walking rate of the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "WalkRate")]
     pub walk_rate: Option<NwValue<i32>>,
+
+    // The willpower bonus of the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "willbonus")]
     pub will_bonus: Option<NwValue<i32>>,
+
+    // The new wings ID associated with the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "Wings_New")]
     pub wings_new: Option<NwValue<i32>>,
+
+    // The wisdom attribute value of the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "Wis")]
     pub wisdom: Option<NwValue<i32>>,
 }
 
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Class {
+    /// Identifier for the struct (optional)
```

Comment:
Don't think I like this (optional) bit, the type implies that. No need to repeat ourselves.

Reply:
(optional) removed in commit https://github.com/milliorn/rs_nwn_nasher_types/pull/9/commits/e5f165459f4bb4a96293e4258ede07815769d18e